sphinx_js/doclets.py

```diff
@@ -45,8 +48,17 @@ def gather_doclets(app):
                 d)
         except PathTaken as conflict:
             conflicts.append(conflict.segments)
+        except ParseError:
+            if not app.config.sphinx_js_lax:
+                raise
+            else:
+                logger.warning('Could not parse correctly %s' % d)
```

Comment:
I'd like to have tests for this output, since formatting a rich datatype like an entire doclet can be error-prone.

Reply:
I added tests and since the formatting was already done in PathsTaken, I piggy backed on it ;)

Reply:
You've added tests for conflict errors, but I was talking about parse errors. :-)

Reply:
You've added tests for conflict errors, but I was talking about parse errors. :-)

Reply:
I added tests for parse errors :)